exonum/src/runtime/dispatcher/mod.rs

```diff
@@ -220,6 +220,20 @@ impl Dispatcher {
             .map_err(From::from)
     }
 
+    /// Initiates stopping of an existing service instance in the blockchain. The stopping
+    /// service is active (i.e., does process transactions or `after_transactions` hook)
+    /// until the block built on top of the provided `fork` is committed.
+    ///
+    /// This method should be called for the exact context passed to the runtime.
```

Comment:
This is a copy-paste artifact, isn't it? There is no context involved in the call.